core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -561,7 +561,7 @@ object TestUtils extends Logging {
                   replicationFactor: Int = 1,
                   servers: Seq[KafkaBroker],
                   topicConfig: Properties = new Properties): scala.collection.immutable.Map[Int, Int] = {
-    val adminZkClient = new AdminZkClient(zkClient)
+    val adminZkClient = new AdminZkClient(zkClient, None)
```

Comment:
do we need this change? the default parameter is already set to None.

(same for other change in this file)